src/project-css/app/Jobs/OptimizeSearch.php

```diff
@@ -0,0 +1,52 @@
+<?php
+/**
+ * @author Tracy A McCormick <tam0013@mail.wvu.edu>
+ */
+
+namespace App\Jobs;
+
+use App\Adapters\UpdateSearchAdapter;
+use Illuminate\Bus\Queueable;
+use Illuminate\Http\Request;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Support\Facades\DB;
+
+use Log;
+
+class OptimizeSearch implements ShouldQueue
+{
+    use InteractsWithQueue, Queueable, SerializesModels;
+
+    private $tblNme;
+    private $skipCount;
+    private $chunkSize;
+    private $records;  
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct($tblNme, $skipCount, $chunkSize)
+    {
+        $this->tblNme = $tblNme;
+        $this->skipCount = $skipCount;
+        $this->chunkSize = $chunkSize;
+    } 
+
+    /**
+     * Execute the job.
+     */
```

Comment:
Needs Proper Documentation: 
Use full PHP Documentation Standards.
Be clear on what the parameters of the function are as well as what the return values are supposed to be.

Describe what job it executes and how the job is handled.